crates/swc_ecma_minifier/src/compress/optimize/util.rs

```diff
@@ -498,6 +500,15 @@ impl<'a> NormalMultiReplacer<'a> {
 
         e.visit_mut_children_with(self);
 
+        let e = if self.should_consume {
```

Comment:
Can we branch bit early to avoid `remove()` + `insert()` for `!self.should_consume` cases?